a3/terraform/modules/cluster/gke-beta/scripts/gke_cluster.sh

```diff
@@ -29,31 +29,31 @@ gke_cluster::create () {
         return 1
     } >&2
 
+    echo "Updating default subnet" >&2
+    gcloud compute networks subnets update default \
```

Comment:
qq: what is the expectation of this change? do we wanna merge it into the main branch as an official release or just a feature branch to ease out the development cycle of our team?

if it is the former, do we also wanna parameterize the name of the control subnet - the default VPC for eth0? Otherwise this script can only be ran once on a specified project.

Reply:
You are right Yufan. I want to merge it to main but before that I want to fix the secondary IP range in our network. The plan is 
1. If we are creating a new VPC for eth0 the we will create the secondary IP range for pod and service and pass those along.
2. If we are using existing VPC for eth0 (like default) we need to update the subnet to create the secondary IP ranges.

I am still trying to figure out how to do #2. I will send out a separate PR for doing that.

Reply:
If the plan is to make it to main, perhaps make the CUSTOM CONTAINERD an optional flag? So far only external customers in Astrophel got whitelisted to use that image so other external customers would fail on this cluster / node pool creation step.

Reply:
In the latest iteration I actually removed `CUSTOM CONTAINERD`. Since the understanding is, we should not expose custom COS in the CPT tool. The gke-beta is expected to work end to end after new GKE version is rolled out.